doc/buildWithDocker.md

```diff
@@ -44,19 +44,19 @@ docker run --rm -it -v $(pwd):/sources --user 1234:1234 infinitime-build
 
 ## Using the image from Docker Hub
 
-The image is available via Docker Hub for both the amd64 and arm64v8 architectures at [pfeerick/infinitime-build](https://hub.docker.com/r/pfeerick/infinitime-build).
+The image is available via Docker Hub for both the amd64 and arm64v8 architectures at [jf002/infinitime-build](https://hub.docker.com/repository/docker/jf002/infinitime-build).
 
 It can be pulled (downloaded) using the following command:
 
 ```bash
-docker pull pfeerick/infinitime-build
+docker pull jf002/infinitime-build
 ```
 
 The default `latest` tag *should* automatically identify the correct image architecture, but if for some reason Docker does not, you can specify it manually:
 
-* For AMD64 (x86_64) systems: `docker pull pfeerick/infinitime-build:amd64`
+* For AMD64 (x86_64) systems: `docker pull jf002/infinitime-build:amd64`
```

Comment:
According to https://stackoverflow.com/questions/60114854/pull-docker-image-for-different-architecture the `--platform` parameter is now the easiest way to explicitly specify the architecture

```sh
docker pull --platform linux/amd64 jf002/infinitime-build
```

```suggestion
* For AMD64 (x86_64) systems: `docker pull --platform linux/amd64 jf002/infinitime-build`
```

Reply:
Thanks for the suggestion. Now, is that parameter really needed? I should detect the platform automatically if it's not specified, I think.

Reply:
no it is not needed. Like stated in the paragraph above the `--platform` parameter can be used to specify the architecture manually if the auto-detect wouldn't work. I just updated the command stated for this purpose to be a bit more up to date. 

I wanted to change the provided commands, because I didn't like the way the tag is used to specify the architecture in those example commands (works, but feels wrong for me). The tag should be used for a version of the package (like for the ubuntu package it defines the release, like `ubuntu:jammy` or `ubuntu:focal`)

Reply:
Thanks, I've just pushed those changes!